apps/protocol-frontend/src/utils/pluralize.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns the number + the plural form of the countable word.
+ *
+ * @param countable A **singular** word to be in plural.
+ * @param n The total number of items.
+ * @param plural Another form of plural if the countable word
+ *                have irregular plural form, e.x: criterion -> criteria
+ */
+const pluralize = (countable: string, n: number, plural?: string): string =>
```

Comment:
Great addition. Love the abstraction for this

Reply:
Thanks! was things for more general implementation but Keating suggested external library.